edx/edx-ora2
# EDUCATOR-5433: clear validation errors on select change

**TL;DR -** Clear "File Types Whitelist" errors when the "File Upload Types" select changes

**What changed?**

- I don't understand what the original thing was doing... I mean, I know what it was literally doing, but I couldn't find `is-shown` anywhere in CSS files or anything. I don't know why that was being set. It seemed to do nothing, and even when calling clearValidationErrors, it still showed up. That was the main change here.

**Testing Instructions**
- go to studio, into an ORA, and into the ORA settings
- enable file upload responses
- "File Upload Types" -> Custom
- clear the box. hit save, there should be an error on the field
- Change File Upload Types to another option. The errors should no longer appear

**Developer Checklist**

JIRA: [EDUCATOR-5433](https://openedx.atlassian.net/browse/EDUCATOR-5433)

FIY: @edx/masters-devs-gta
